### ISSUE TITLE
Make readme beautiful and more user-friendly

### ISSUE THREAD

**Issue by sr6033** [sr6033/lterm#4]
Try to add more instructions on how to contribute to the project. Also if possible, keep a slight sense of humour in your writing and some beautiful emoticons and buttons.

**Comment by Logan1x** [comment]
I would like to work on this.

**Comment by sr6033** [comment]
Keep a slight humour if possible ;)

**Comment by Logan1x** [comment]
will try to....

**Comment by Logan1x** [comment]
do you have a list of instruction which is not included in the present version of readme? @sr6033

**Comment by sr6033** [comment]
No. No more instructions. However, if you find any difficulty in understanding and would like to add more descriptions, feel free to send in a PR. I will review it.
